config/default.schema.json

```diff
@@ -1588,6 +1593,7 @@
 				}
 			},
 			"default": {
+				"WEBSOCKET_URL": "ws://localhost:3345",
```

Comment:
Sollte der Default-Wert nicht mehr so `wss://{{ DOMAIN }}/tldraw-server` aussehen? Siehe ansible/roles/schulcloud-server-core/templates/configmap.yml.j2 .

Reply:
Oder vielleicht ist auch der "obere" Wert der falsche?

Reply:
In ansible wird so wss://{{ DOMAIN }}/tldraw-server verwendet weil er zu richtige pod routen soll und local reicht wenn wir Port verwenden